pandas/util/_decorators.py

```diff
@@ -294,14 +296,11 @@ def decorate(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
             arguments = _format_argument_list(allow_args)
-            if isinstance(allow_args, (list, tuple)):
-                num_allow_args = len(allow_args)
-            else:
-                num_allow_args = allow_args
+            num_allow_args = len(allow_args)
```

Comment:
ndb, but this and the line above could be moved to the outer scope.

Reply:
indeed - have fixed, thanks!